### PR TITLE
fix: set zIndex to select input style

### PR DESCRIPTION
**Pull request by gaeVG**

### CODE REVIEW

File: src/components/Dashboard/FormDashboard/FormReference.jsx [hunk 1]
```diff
@@ -338,6 +338,9 @@ export default function FormReference({ category, reference }) {
           options={countries}
           className="form-input"
           defaultInputValue={reference.country ? reference.country : ""}
+          styles={{
+            menu: provided => ({ ...provided, zIndex: 2 })
+          }}
```
Comment (Lauraillouz) [inline src/components/Dashboard/FormDashboard/FormReference.jsx:343]: why inline style?

File: src/components/Dashboard/FormDashboard/FormReference.jsx [hunk 2]
```diff
@@ -336,7 +336,7 @@ export default function FormReference({ category, reference }) {
         <Select
           onChange={(e) => setCountry(e.label)}
           options={countries}
-          className="form-input"
+          className="form-input countries-input"
           defaultInputValue={reference.country ? reference.country : ""}
```
Comment (Lauraillouz) [inline src/components/Dashboard/FormDashboard/FormReference.jsx:340]: no need for a new class, you could use a modifier "countries" and add "form-input_countries"

File: src/styles/sass/sasscomponents/_addReference.scss [hunk 3]
```diff
@@ -20,3 +20,7 @@
     }
   }
 }
+
+.countries-input {
+  z-index: 2;
+}
```
Comment (Lauraillouz) [inline src/styles/sass/sasscomponents/_addReference.scss:26]: and you could nest that in form-input like:
&_countries { z-index: 2;}

**Comment by gaeVG** [comment]
@Lauraillouz the names of the classes have been modified, it should be good now !

**Pull request merged**